statsmodels/discrete/tests/test_discrete.py

```diff
@@ -2012,6 +2038,7 @@ def setup_class(cls):
                                                                  disp=0)
         res2 = RandHIE()
         res2.negativebinomial_nb1_bfgs()
+        #res2.phat = np.exp(res2.fittedvalues)
```

Comment:
Why?

Reply:
one of my trials to retrofit the results, can be deleted again
It turned out it's simpler to add the exp to test_predict, as long as only count models are missing phat.